src/components/ProductList/ProductItem/ProductItem.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function ProductItem() {
+  return (
+    <Item>
+      <ImgContainer>
+        <img alt="real estate" src="http://placehold.it/100" />
+      </ImgContainer>
+      <ItemInfo>
+        <ItemPercent>
+          <i className="fas fa-chart-pie"></i>
+          <span>64% 모집</span>
+        </ItemPercent>
+        <p>주거안정 357호 세종 새뜸1단지 메이저시티푸르지오</p>
+        <ItemNotice>
+          <b>8.5%</b>
+          <span>12개월 · 20,000만원 모집</span>
+        </ItemNotice>
+      </ItemInfo>
+    </Item>
+  );
+}
+
+export default ProductItem;
+
+const Item = styled.li`
+  cursor: pointer;
+
+  .imgContainer {
+  }
+`;
+
+const ImgContainer = styled.div`
+  margin-bottom: 20px;
+  width: 100%;
```

Comment:
```suggestion
const ImgContainer = styled.div`
  margin-bottom: 20px;
```
이미 가로 100%인 요소입니다. 확인해보시고 불필요하면 지워주세요